allensdk/internal/pipeline_modules/run_neuropil_correction.py

```diff
@@ -227,7 +227,7 @@ def main():
         hf.create_dataset("r", data=r_list)
         hf.create_dataset("RMSE", data=RMSE_list)
         hf.create_dataset("FC", data=corrected, compression="gzip")
-        hf.create_dataset("roi_names", data=roi_names)
+        hf.create_dataset("roi_names", data=roi_names.astype(np.string_))
```

Comment:
Just to double check, some downstream functionality (presumably in python 2) expects this to be a bytestring?

Reply:
DF/F computation uses this file.
/shared/bioapps/infoapps/lims2_modules/CAM/dff/compute_dff.py

Reply:
That example was an old run.  DF/F has been modernized:
/allen/aibs/technology/conda/production/allensdk_py36 python -m allensdk.internal.pipeline_modules.run_dff_computation

Reply:
as far as I can tell, the downstream components that consume these outputs are:
1. the dff calculator, which passes roi_names through: https://github.com/AllenInstitute/AllenSDK/blob/1e8950a31a987f10267765ab34e66cf3a4e54128/allensdk/internal/pipeline_modules/run_dff_computation.py#L43
2. the behavior ophys nwb writer, which ignores them:
https://github.com/AllenInstitute/AllenSDK/blob/1e8950a31a987f10267765ab34e66cf3a4e54128/allensdk/internal/api/behavior_ophys_api.py#L115
3. the legacy cam nwb creator, which seems to just want these data to match between the neuropil corrected and uncorrected files. Then it ignores them:
http://stash.corp.alleninstitute.org/projects/INF/repos/lims2_modules/browse/CAM/cam_nwb/nwb_creator.py#261

These are actually integers, so it is baffling to me that they are being stored as strings, but that seems like a larger set of changes.

Reply:
I think Nick just added this recently because users were complaining that they didn't know what id each data point went with.  I do not believe the "roi_names" are used.